pkg/systemd/services.jsx

```diff
@@ -869,7 +869,7 @@ const ServicesPageFilters = ({
                  clearAllFilters={onClearAllFilters}
                  className="pf-m-sticky-top ct-compact services-toolbar"
                  id="services-toolbar"
-                 numberOfFiltersText={n => cockpit.format("$0 filters applied")}>
+                 numberOfFiltersText={n => cockpit.format(_("$0 filters applied"), n)}>
```

Comment:
This added line is not executed by any test.

Reply:
Nice catch!